tests/units/offline/test_utils.py

```diff
@@ -140,6 +142,29 @@ def test_get_pose_matrix() -> None:
         get_pose_matrix([0.1, 0.2, 0.1], [-20, -90, -50, 10])
 
 
+@pytest.mark.offline
+def test_quaternion_from_euler_angles() -> None:
+    q1 = quaternion_from_euler_angles(0, 0, 0)
+    assert q1 == Quaternion(1, 0, 0, 0)
+    q2 = quaternion_from_euler_angles(90, 0, 0)
+    assert q2 == Quaternion(axis=[1, 0, 0], angle=np.pi / 2)
+    q3 = quaternion_from_euler_angles(0, 90, 0)
+    assert q3 == Quaternion(axis=[0, 1, 0], angle=np.pi / 2)
+    q4 = quaternion_from_euler_angles(0, 0, 90)
+    assert q4 == Quaternion(axis=[0, 0, 1], angle=np.pi / 2)
+    q5 = quaternion_from_euler_angles(45, 45, 45).normalised
+    expected_quat = Quaternion(axis=[0.57735027, 0.57735027, 0.57735027], angle=np.pi / 4).normalised
```

Comment:
q5 = quaternion_from_euler_angles(0, 0 0).normalised also pass the test

Why not:
```
    q5_ref = Quaternion(axis=[0.679, 0.281, 0.679], angle=np.deg2rad(85.8))
    unit_quat = Quaternion()
    expected_unit = q5 * q5_ref.inverse
    assert np.allclose(unit_quat.elements, expected_unit.elements, atol=1e-3)
```
A quaternion multiply by its inverse will give a unit quaternion